src/test/java/io/strimzi/kafka/bridge/mqtt/mapper/MqttKafkaRegexMapperTest.java

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.mapper;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.PatternSyntaxException;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertThrows;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.CoreMatchers.startsWith;
+
+/**
+ * Unit tests for {@link MqttKafkaSimpleMapper}
+ */
+public class MqttKafkaRegexMapperTest {
+
+
+    /**
+     * Test for default topic.
+     * If the MQTT topic does not match any of the mapping rules, the default topic is used.
+     * E.g. if the Topic Mapping Rules is empty, the default topic is used by default.
+     */
+    @Test
+    public void testDefaultTopic() {
+        List<MappingRule> rules = new ArrayList<>();
+
+        MqttKafkaRegexMapper mapper = new MqttKafkaRegexMapper(rules);
+
+        assertThat("Should use the default topic when no mapping pattern matches.",
+                mapper.map("sensor/temperature").kafkaTopic(), is(MqttKafkaSimpleMapper.DEFAULT_KAFKA_TOPIC));
+
+        assertThat("The key for the default topic should be null",
+                mapper.map("sensor/temperature").kafkaKey(), nullValue());
```

Comment:
I was actually doing this, see the first commit. 
but as the number of tests increases, to avoid a lot of lines, I decided to call it when we need to test.

Reply:
Yeah but did you get the current is somehow wrong? You are doing testing on two different instances of MappingResult because you call map two times. Back to the initial tests then. More lines but correct ones :-)